spark_pipeline_framework/transformers/send_automapper_to_fhir/v1/automapper_to_fhir_transformer.py

```diff
@@ -291,6 +310,17 @@ def _transform(self, df: DataFrame) -> DataFrame:
                             mode=mode,
                             run_synchronously=run_synchronously,
                             num_partitions=parameters.get("num_partitions"),
+                            sort={
+                                "column_for_sorting": sort_data.get(  # type: ignore
+                                    "column_for_sorting"
+                                ),
+                                "drop_column": sort_data.get("drop_column"),  # type: ignore
+                                "partition_by_column_name": sort_data.get(  # type: ignore
+                                    "partition_by_column_name"
+                                ),
+                            }
```

Comment:
avoid type:ignore

Reply:
I've added these because I was getting an error : `Item "None" of "dict[str, Any] | None" has no attribute "get` but sort_data dict existence has already been checked in need_sorting condition above.